algosP.yaml

```diff
@@ -0,0 +1,67 @@
+float:
+  any:
+    bruteforce:
+      docker-tag: ann-benchmarks-sklearn
+      module: ann_benchmarks.algorithms.bruteforce
+      constructor: BruteForce
+      base-args: ["@metric"]
+      run-groups:
+        empty:
+          args: []
+    bruteforce-blas:
+      docker-tag: ann-benchmarks-sklearn
+      module: ann_benchmarks.algorithms.bruteforce
+      constructor: BruteForceBLAS
+      base-args: ["@metric"]
+      run-groups:
+        empty:
+          args: []
+  angular:
+    pp-bruteforce-lo:
+      module: ann_benchmarks.algorithms.subprocess
+      docker-tag: ann-benchmarks-subprocess
```

Comment:
`install/Dockerfile.subprocess` is missing though. Did you forget to include that file?

Reply:
Good point; we never actually *wrote* that file! `ann_benchmarks/algorithms/definitions.py` now requires that every algorithm have a `docker-tag` property, so I just put a value in there to identify this group.